src/main/java/ScrabbleGame.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class ScrabbleGame {
+
+    List<String> playedWords = new ArrayList<>();
+    List<Character> tiles = new ArrayList<>();
+    List<Character> tilesTesting = new ArrayList<>();
```

Comment:
There's probably a way to do this without having this List as a member variable.